phpunit/block-supports/typography-test.php

```diff
@@ -283,6 +283,111 @@ public function test_should_generate_classname_for_font_family() {
 		$this->assertSame( $expected, $actual );
 	}
 
+	/**
+	 * Tests that stabilized typography supports will also apply to blocks using
+	 * the experimental syntax, for backwards compatibility with existing blocks.
+	 *
+	 * @covers ::wp_apply_typography_support
```

Comment:
```suggestion
	 * @covers ::gutenberg_apply_typography_support
```

Should this be the Gutenberg function while the test is in this plugin?